src/components/typography/Heading.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from "classnames";
+import { nullValuesAsUndefined } from "null-as-undefined";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/core/_typography.scss";
+
+/**
+ * The level of heading for {@link Heading}.
+ */
+export enum HeadingLevels {
+  H1 = 1,
+  H2 = 2,
+  H3 = 3,
+  H4 = 4,
+  H5 = 5,
+  H6 = 6
+}
+
+/**
+ * The proptypes for {@link Heading}.
+ */
+export interface HeadingProps {
+  id?: string | null;
+  className?: string | null;
+  level: HeadingLevels;
+  /**
+   * Style large with a light font.
+   *
+   * See `.lbh-heading-large-light` in `lbh-frontend` for the implementation
+   * details.
+   */
+  largeLight?: boolean | null;
+  /**
+   * The heading text.
+   */
+  children: React.ReactNode;
+}
+
+/**
+ * A component that creates a heading tag.
+ */
+export const Heading: React.FunctionComponent<HeadingProps> = (
+  props: HeadingProps
+): JSX.Element => {
+  const { id, level, largeLight, children } = nullValuesAsUndefined(props);
+
+  const HeadingLevel = `h${level}` as "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
```suggestion
  const H = `h${level}` as "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

No need to change it, but I think this is clearer in some ways. It's not the level, it's the tag itself. For future reference.